demos/demo-react-ts/src/components/screens/KeypadScreen.tsx

```diff
@@ -95,21 +108,35 @@ function KeypadScreen({
       callStartTime,
     });
     startTimer(callStartTime);
+    setDirection("OUTBOUND");
     handleNextScreen();
-  }, [cti, dialNumber, handleNextScreen, startTimer]);
+  }, [cti, dialNumber, handleNextScreen, startTimer, setDirection]);
+
+  const handleTriggerIncomingCall = useCallback(
+    (incomingCallNumber: string) => {
+      cti.incomingCall({
+        createEngagement: true,
+        fromNumber: incomingCallNumber,
+        toNumber: fromNumber, // dialNumber can be empty and passing empty number can make this fail
```

Comment:
We can remove the comment here since it makes sense to only use dialNumber for outbound